charts/fluent-bit/README.md

```diff
@@ -0,0 +1 @@
+# fluent-bit
```

Comment:
add the README file with some proper description, u may refer to the official fluent-bit chart.